### ISSUE TITLE
[LogBlock] PlayerInteractEvent error

### ISSUE THREAD

**Issue by Victor9401** [keepcalm/BukkitForge#368]
I've got a very strange error with my BukkitForge server
This problem is very weird, because i can only reproduce it on non Notch based Minecraft launchers, yes launchers. If you use original or patched original launcher, it will work fine. I am getting this errors on block place/break with adminium (LB toolblock, also same with tool). Not only LB does not work, i've tested LWC, it also got same problems.
For example you can spawn chest, open it, break it normally, but if you want, for example, to private chest, you will have an error in chat and become unable to break or create blocks, but chest will become private.
Same is with LB you will simply spam world and console with adminium and errors.

Mods and plugins installed:
Tested on clean server only with LogBlock or LWC

Another thing, LB on MCPC or MCPC+ works fine, so i think trouble is in BF

Error log for LB:
[SEVERE] [BukkitForge] Could not pass event PlayerInteractEvent to LogBlock v1.61
org.bukkit.event.EventException
    at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:397)
    at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:66)
    at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:477)
    at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:462)
    at keepcalm.mods.bukkit.forgeHandler.ForgeEventHandler.interactEvent(ForgeEventHandler.java:464)
    at net.minecraftforge.event.ASMEventHandler_34_ForgeEventHandler_interactEvent_PlayerInteractEvent.invoke(.dynamic)
    at net.minecraftforge.event.ASMEventHandler.invoke(ASMEventHandler.java:35)
    at net.minecraftforge.event.EventBus.post(EventBus.java:103)
    at net.minecraftforge.event.ForgeEventFactory.onPlayerInteract(ForgeEventFactory.java:34)
    at ir.a(ItemInWorldManager.java:159)
    at iv.a(NetServerHandler.java:509)
    at ei.a(SourceFile:46)
    at cg.b(TcpConnection.java:458)
    at iv.d(NetServerHandler.java:136)
    at iw.b(NetworkListenThread.java:57)
    at ht.b(SourceFile:30)
    at net.minecraft.server.MinecraftServer.r(MinecraftServer.java:703)
    at ho.r(DedicatedServer.java:270)
    at net.minecraft.server.MinecraftServer.q(MinecraftServer.java:599)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:497)
    at fy.run(SourceFile:849)
Caused by: java.lang.NullPointerException
    at de.diddiz.LogBlock.listeners.ToolListener.onPlayerInteract(ToolListener.java:49)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:395)
    ... 20 more

**Comment by Victor9401** [comment]
Any one?

**Comment by gravypod** [comment]
We tested HawkEye the other day and logblock, both seemed to work fine. Any idea how to cause this?

**Comment by SpoonsJTD** [comment]
Please recheck this.

**Comment by charlieTheBotDev** [comment]
I've just added LogBlock to my server, I tried it to use it and it actually worked! I was actually quite shocked, it didn't throw any errors at all and logged the blocks I was placing.

I was happy right up until I tried it with some non-vanilla blocks. It doesn't seem to detect them at all. I've tried adding them to the materials.yml (namely 1311:1, Fir wood planks and 2001, Copper ore) but it didn't seem to help at all. On the plus side, the player interact errors seem to have gone :)

I'm running:
BukkitForge build 255
LogBlock 1.7
